From 758ebee56fa8fd617121e350ecf1289387b73f5d Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Mon, 7 Sep 2020 11:45:03 -0400
Subject: [PATCH] test: Install required helper files

During the conversion from Autotools to Meson, this part was accidentally
lost, which makes the test backend not work quite right.

This change installs the missing files for the test_spawn part of the
test backend.
---
 backends/test/helpers/meson.build | 4 ++++
 backends/test/meson.build         | 2 ++
 2 files changed, 6 insertions(+)
 create mode 100644 backends/test/helpers/meson.build

diff --git a/backends/test/helpers/meson.build b/backends/test/helpers/meson.build
new file mode 100644
index 000000000..486b7620f
--- /dev/null
+++ b/backends/test/helpers/meson.build
@@ -0,0 +1,4 @@
+install_data(
+  'search-name.sh',
+  install_dir: join_paths(get_option('datadir'), 'PackageKit', 'helpers', 'test_spawn'),
+)
diff --git a/backends/test/meson.build b/backends/test/meson.build
index eba7dcdaa..bdf040c05 100644
--- a/backends/test/meson.build
+++ b/backends/test/meson.build
@@ -1,3 +1,5 @@
+subdir('helpers')
+
 shared_module(
   'pk_backend_test_fail',
   'pk-backend-test-fail.c',
-- 
2.26.2

